test/e2e/components/vl-map-modify-action.js

```diff
@@ -0,0 +1,17 @@
+const VlMapAction = require('./vl-map-action');
+const VlMapLayer = require('./vl-map-layer');
+
+
+class VlMapModifyAction extends VlMapAction {
+  async getLayer() {
+    return new VlMapLayer(this.driver, await this.driver.executeScript('return arguments[0].parentElement', this));
```

Comment:
Dit gaat niet meer echt kloppen icm. de nieuwe layers.

Reply:
ja maar uwe PR zit ook nog niet in de master he @SanderKleykens  ...

Reply:
Dat weet ik, maar als je merged naar bv voortoets-demo branch (en uiteraard ook master) ga je tegen dit probleem botsen, is maar wat ik wil zeggen.

Reply:
weet niet of ik al achter zit op master, normaal had ik vrijdag nog eens gerebased. Zal dit meteen nog eens doen.

Reply:
Neenee, het zit nog niet op master zoals @pascalriquier zegt. Het is gewoon dat de branch met WFS/WMS/WMTS layers de VlMapLayer "abstract" maakt, maar dat zit nog niet in master.
Maar als je dit gaat toevoegen op de voortoets-demo page, ga je moeten mergen en ga je hier wel tegenlopen.